File: Aula19_19_11_2020/Cs/Ex1.cs
using System;

namespace Cs
{
    class Ex1
    {
        static void Main(string[] args)
        {
            int i, num, f;

            Console.WriteLine("Digite o número: ");
            num = Convert.ToInt32(Console.ReadLine());
            
            if (num == 0){
                Console.WriteLine("O fatorial de 0 é 0");
            }
            else
            {
                f = 1;
                i = 1;
            
                while(i <= num)
                {
                    f *= i;
                    i++;
                }
                Console.WriteLine($"Fatorial é = {f}");
            }

        }
    }
}

File: Aula19_19_11_2020/Py/Ex1.py
num = int(input("Digite o número: "))

if num == 0:
    print("O fatorial de 0 é 0")
else:
    f = 1
    i = 1
while i <= num:
    f *= i
    i += 1
print("Fatorial é {}".format(f))